dll/shellext/fontext/CFontExt.cpp

```diff
@@ -568,7 +564,7 @@ HRESULT CFontExt::DoInstallFontFile(LPCWSTR pszFontPath, LPCWSTR pszFontsDir, HK
     LPCWSTR pszFileTitle = PathFindFileName(pszFontPath);
 
     WCHAR szFontName[512];
-    if (!DoGetFontTitle(pszFontPath, szFontName))
+    if (!DoGetFontTitle(pszFontPath, szFontName, sizeof(szFontName)))
```

Comment:
can we improve this to use CString instead of a fixed size buffer?

Reply:
OK.